syntax = "proto3";

package spy_pipe_pkg;

service SpyPipeGRPC {
    rpc SendData(stream Data) returns (stream Data) {}
}

message Data {
    bytes  payload = 1;
    string control = 2;
}


